elide-core/src/main/java/com/yahoo/elide/core/pagination/Pagination.java

```diff
@@ -80,9 +89,18 @@ public static Pagination parseQueryParams(final MultivaluedMap<String, String> q
                         } catch (ClassCastException e) {
                             throw new InvalidValueException("page values must be integers");
                         }
+                    } else if (queryParamKey.equals(PAGE_TOTAL_KEY)) {
+                        final String value = paramEntry.getValue().get(0);
+                        if (value.equals("true")) {
```

Comment:
Is there a reason we care about the values here? I think it would probably be ok to check for the existence of this parameter (regardless of its value) to enable this feature-- by default it should probably stay disabled anyway.